src/main/java/Storage.java

```diff
@@ -0,0 +1,78 @@
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+
+    private final String filePath;
+
+    /**
+     * constructor for Storage object
+     * @param filePath path of the data file
+     */
+     public Storage(String filePath){
+         this.filePath = filePath;
+     }
+
+    /**
+     * method to read from hard disk
+     * @return reads the txt file amd converts it into a list of taskd
```

Comment:
Not sure if I like the style of the JavaDoc comments. You can consider following the style set out in the coding standard: Start off the summary immediately with a verb like "Returns, Reads, Adds... etc". Additionally, should the text in the return annotation, specifically the part on "reads the txt file", be part of the method summary instead?